bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/ModelConverters.scala

```diff
@@ -194,7 +198,13 @@ object ModelConverters {
         case t:HmsDegTarget => Some(t.getRedshift)
         case _              => None
       }
-      SiderealTarget(name, coordinates, pm, z, px, mags)
+      SiderealTarget.empty.copy(
```

Comment:
Do `SpectralDistribution` or `SpatialProfile` have any sense on the old model
Somewhat I'd like in here to use the full constructor so if it changes like in this PR you must remember to update the conversion, otherwise this will compile cleanly